token.sh

```diff
@@ -14,22 +13,37 @@ API_VERSION=v3
 API_HEADER="Accept: application/vnd.github.${API_VERSION}+json"
 AUTH_HEADER="Authorization: token ${ACCESS_TOKEN}"
 
-REPO_URL=${REPO_URL:-${URI}}
+if [[ ${RUNNER_SCOPE} == "repo" ]]; then
+  [[ -z ${REPO_URL} ]] && ( echo "REPO_URL required for repo runners"; exit 1 )
+else
+  REPO_URL=${URI}
+fi
+
 _PROTO="$(echo "${REPO_URL}" | grep :// | sed -e's,^\(.*://\).*,\1,g')"
 # shellcheck disable=SC2116
 _URL="$(echo "${REPO_URL/${_PROTO}/}")"
 _PATH="$(echo "${_URL}" | grep / | cut -d/ -f2-)"
 _ACCOUNT="$(echo "${_PATH}" | cut -d/ -f1)"
 _REPO="$(echo "${_PATH}" | cut -d/ -f2)"
 
-_FULL_URL="${URI}/repos/${_ACCOUNT}/${_REPO}/actions/runners/registration-token"
-if [[ ${_ORG_RUNNER} == "true" ]]; then
-  [[ -z ${ORG_NAME} ]] && ( echo "ORG_NAME required for org runners"; exit 1 )
-  _FULL_URL="${URI}/orgs/${ORG_NAME}/actions/runners/registration-token"
-  _SHORT_URL="${_PROTO}${_GITHUB_HOST}/${ORG_NAME}"
-else
-  _SHORT_URL=$REPO_URL
-fi
+case ${RUNNER_SCOPE} in
```

Comment:
id like to see if theres a way to shorten the next bit, it seems repetitive from `entrypoint.sh`